programs/ssl/ssl_client2.c

```diff
@@ -363,9 +365,14 @@ int main( void )
 
 #if defined(MBEDTLS_SSL_CONTEXT_SERIALIZATION)
 #define USAGE_SERIALIZATION \
-    "    serialize=%%d        default: 0 (do not serialize/deserialize)\n" \
-    "                        options: 1 (serialize)\n"                    \
-    "                                 2 (serialize with re-initialization)\n"
+    "    serialize=%%d        default: 0 (do not serialize/deserialize)\n"     \
+    "                        options: 1 (serialize)\n"                         \
+    "                                 2 (serialize with re-initialization)\n"  \
+    "    serial_to_file=%%s   The file path to write a serialized connection\n"\
```

Comment:
Could we rename this to "context_file"? "Serial to file" doesn't make very much sense in English.

Reply:
Yes, I know. I wanted to use an abbreviation instead of a full word like for example 'renego' instead of 'renegotiation' :). I changed to 'context_file'